contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,15 +1,13 @@
 * **Objective**:
-    * Increase word-of-mouth growth by:
-      1. Showcasing new features by acting as the newsroom of PostHog
-      2. Being an authority and vital resource on building successful products.
+    * Become an authority and vital resource on building successful products.
 
 * **Goals:**
     * Nail Twitter for content distribution (**Owner:** Andy)
     * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)
     * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)
-    * Create a hub of non-PostHog product engineering content (**Owner:** Lior)
+    * Create a hub of non-PostHog product engineering content (**Owner:** Andy)
```

Comment:
```suggestion
    * Create a hub of non-PostHog product engineering content (**Owner:** Andy & Ian)
```

Reply:
This one can just be me, if that is good with you

Reply:
Yup